### DISCUSSION TITLE
126

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#58] [General]
# 潮流周刊第126期 - 买了个球

封面来源于上周逛宜家时候，走的时候看到有卖绿植的，随手买了个仙人球，还挺有生命力的，希望可以被我养活 😂。

https://weekly.tw93.fun/posts/126-%E4%B9%B0%E4%BA%86%E4%B8%AA%E7%90%83

**Comment by i-mhLee** [comment]
赞
